index.rst

```diff
@@ -1,61 +1,365 @@
+
+IDM-0203
+    Users must be able to see their recent web authentications, at least including timestamp and external authentication provider.
+    Ideally this should include GeoIP information for the IP address, although getting accurate data inexpensively can be challenging so this isn't a firm requirement.
+
+IDM-0204
+    When displaying the list of federated identities associated with the account, the date and time at which that identity was last used to authenticate must be shown alongside.
+
+IDM-0205
+    When displaying the list of user-generated tokens, the date and time at which a user-generated token was last used must be shown alongside the token name.
+
+IDM-0206
+    Users must be notified via email of any change to their linked federated identities or any creation or deletion of a new user-generated token.
+
+Account management
+==================
+
+Status
+------
+
+IDM-1000
+    Accounts that are pending approval (under IDM-0003) can authenticate and see their account status and metadata page, but not access any other part of the Science Platform.
+    They may not create tokens.
+
+IDM-1001
+    Administrators of the Science Platform must be able to freeze accounts.
+    Frozen accounts may be placed in a state where they cannot authenticate at all, or in a state where they can only see their account status and metadata page but no other part of the Science Platform.
+    A reason viewable by other administrators should be associated with a frozen account.
+    Frozen accounts still hold the account name and do not allow it to be reused.
+
+IDM-1002
+    Administrators of the Science Platform must be able to delete accounts.
+    This is normally used for mistakenly-created accounts, not for accounts that were legitimate and active but should no longer be allowed access.
+
+IDM-1003
+    It must be possible to set an expiration date on an account.
+    This can be done by Science Platform administrators, or by the person approving access in the IDM-0003 use case.
+    When the expiration date arrives, the account must be automatically frozen.
+
+IDM-1004
+    Users must be notified via email of upcoming account expiration so that they can investigate renewal options if needed.
+
+Metadata
+--------
+
+IDM-1100
+    A full name must be associated with each account and prepopulated with information from the identity provider.
+    The user must be able to change the full name to anything they wish.
+
+IDM-1101
+    An email address must be associated with each account and prepopulated with information from the identity provider.
+    The user must be able to change the email address to anything they wish, but they must then verify that the email address is valid and owned by them by responding to a challenge sent to that email address.
+    Challenges for an email address must not contain user-provided content so that they cannot be used for spamming purposes.
+
```

Comment:
I've added some requirements for tracking the per-user metadata, including institutional affiliation and user class. I'm not sure I have clear enough requirements for the workflow for reviewing affiliation or the expectations for what affiliation review will look like to turn this into identity management requirements. Is there a document somewhere laying out how this will work, or is it not yet determined?